src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { useContext } from 'react';
+import { TodoContext } from '../Store/TodoContext';
+import { TodoItem } from '../TodoItem/TodoItem';
+import { Status } from '../../types/Status';
+
+export const TodoList: React.FC = () => {
+  const { todos, filterValue } = useContext(TodoContext);
+
+  const filteredTodos = todos.filter(todo => {
+    if (filterValue === Status.Active) {
+      return !todo.completed;
+    }
+
+    if (filterValue === Status.Completed) {
+      return todo.completed === true;
```

Comment:
you forgot to change it here :slightly_smiling_face: :
![image](https://github.com/mate-academy/react_todo-app-loading-todos/assets/86923089/bd3df280-8a31-42f6-bfea-2a166322449b)

Fix it in the next step task